### DISCUSSION TITLE
Uncaught Error when open the app with Vercel

### DISCUSSION THREAD

**Discussion by NirachaMarchett** [neuefische/questions#693] [Web] [answered]
Hi everyone, 

when I open the app via deployment with Vercel, I got these error messages. I'm not sure what's the problem is cause there's no error when open the app with localhost. 

![Screenshot 2023-08-25 at 10 50 58](https://github.com/neuefische/questions/assets/135506311/c5387b09-f5c2-4bcd-ad31-0fe56c10f527)

Seems to be a problem with Verce (Vercel failed to deploy)l, I'm not sure how to fix it, could someone please help? 

here's the link to my PR [Final adjustments](https://github.com/NirachaMarchett/capstone-expiration-tracking-app/pull/33)

Thank you so much!

**Comment by julialoeschel** [comment, marked as answer]
Hey there. 

Your structure in `/pages` is the problem. 
Next Provides the Pages folder [only for React components](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts), which are going to be a page. 

You put in `/pages/favorite`and `/pages/recipe`a `.style.js`file to style your pages. Thats the Problem. Try to find another way to style your pages (about components, Globalstyles or place necessary styles within the page file) 

hope that helps. 
Julia
